src/sql/db_connection_pool/dbconnection/mysqlconn.rs

```diff
@@ -43,6 +44,26 @@ pub struct MySQLConnection {
     pub conn: Arc<Mutex<Conn>>,
 }
 
+impl MySQLConnection {
+    fn column_meta_query_for(tbl: &TableReference) -> String {
+        let q = MySqlDialect {}
+            .identifier_quote_style("") // parameter unimportant for `MySqlDialect`.
```

Comment:
Fixed part 2. For part 1. wanted to keep it non-hardcoded in case the pattern needs to be used elsewhere.

Reply:
Also `column_meta_query_for` is a utility function, just under the `MySQLConnection` implementation. Is there a better spot?

Reply:
For readability, future modifications and other contributors I think it might be beneficial to have it as a separate `to_mysql_quoted_string` fn with a note why `table_reference.to_quoted_string()` can't be used

Reply:
This is wise. I like it.